src/templates/seo-head.js

```diff
@@ -11,14 +12,15 @@ import {
 
 export const nuxtI18nSeo = function () {
   if (
+    !seo ||
     !(VueMeta.hasMetaInfo ? VueMeta.hasMetaInfo(this) : this._hasMetaInfo) ||
     !this.$i18n ||
     !this.$i18n.locale ||
     !this.$i18n.locales ||
     this.$options[COMPONENT_OPTIONS_KEY] === false ||
     (this.$options[COMPONENT_OPTIONS_KEY] && this.$options[COMPONENT_OPTIONS_KEY].seo === false)
   ) {
-    return {}
+    return { htmlAttrs: {} }
```

Comment:
since in the `headMeta` function, if the seo is enabled, we have to ensure that the returned object has the `htmlAttrs` property. 

```JS
 let metaObject
  if (seo) {
    metaObject = nuxtI18nSeo.call(this)
  } else {
    metaObject = { htmlAttrs: {} }
  }
```

but yeah, it seems a little bit ambiguous this way, i think it might be better to revert back and return an empty object as before. And instead, check the existence of the property in the `headMeta` as following maybe: 

```JS
  let metaObject = {htmlAttrs: {}}
  if (seo) {
    const seoMeta = nuxtI18nSeo.call(this)
    if (Object.keys(seoMeta).length !== 0){
      metaObject = seoMeta;
    }
  }
```
what do you think?

Reply:
For the least amount of duplication maybe:

```js
  let metaObject = {}
  if (seo) {
    metaObject = nuxtI18nSeo.call(this)
  }
  if (!metaObject.htmlAttrs) {
     metaObject.htmlAttrs = {};
  }
```

But if you address my other comment then maybe this whole file won't be necessary.

Reply:
looks much better. Yes it won't be necessary at all. **if** we decided later to go with this approach, i will then commit the changes.